recipes/sqlitecpp/all/conanfile.py

```diff
@@ -41,9 +43,17 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
+        if self.options.has_codec:
+            self.options["sqlcipher"].enable_column_metadata = True
 
     def requirements(self):
-        self.requires("sqlite3/3.45.0")
+        if self.options.has_codec:
+            if Version(self.version) < "3.3.1":
+                raise ConanInvalidConfiguration("Using SQLCipher with this recipe is only available from version 3.3.1")
+            else:
+                self.requires("sqlcipher/[>=4.5.6]")
+        else:
+            self.requires("sqlite3/3.45.0")
```

Comment:
Why is sqlite3 always required now? Also, please, do not use version range for sqlcipher: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges